### DISCUSSION TITLE
Dead links

### DISCUSSION THREAD

**Discussion by dgossa** [inventree/InvenTree#7055] [General]
I cannot download the 3 required files to start the production server install. The links seem to be dead and /docker forlder does not exist in the source code. Please let me know if I'm missing something.

https://docs.inventree.org/en/stable/start/docker_install/#before-you-start
Under Docker Installation -> Required Files

**Comment by SchrodingersGat** [comment]
@dgossa thanks for reporting, these links have now  been fixed.
